src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import React from 'react';
+
+type PaginationProps = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const pages = [];
+
+  for (let i = 1; i <= totalPages; i++) {
+    pages.push(i);
+  }
+
+  const startIndex = (currentPage - 1) * perPage;
+  const endIndex = Math.min(startIndex + perPage - 1, total - 1);
+
+  const itemsOnCurrentPage = [];
+
+  for (let i = startIndex; i <= endIndex; i++) {
+    itemsOnCurrentPage.push(i);
+  }
+
+  return (
+    <div>
+      <ul className="pagination">
+        <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled="true"
+            onClick={() => onPageChange(currentPage - 1)}
```

Comment:
Consider improving you disabled logic for navigation pages

https://www.youtube.com/watch?v=MxXZCD0XF2k